core/lib/dal/sqlx-data.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is it required?

Reply:
Nope it isn't, thought it was deleted though, removing it asap.